script/audit_deps.py

```diff
@@ -14,43 +14,37 @@
 import subprocess
 import sys
 
+import requests
+
 from deps_config import RUST_DEPS_PACKAGE_VERSION
 
+# pylint: disable=inconsistent-return-statements
+def get_remote_audit_config(
+        url = "https://raw.githubusercontent.com/brave/audit-config/main/config.json",
+        retry = 3):
+    """Fetch additional audit configuration"""
+    for i in range(retry):
+        try:
+            return json.loads(requests.get(url).text)
```

Comment:
requests has `response.json()` which should be more idiomatic
https://requests.readthedocs.io/en/latest/user/quickstart/#json-response-content

Reply:
Thanks, good suggestion. Fixed.